### ISSUE TITLE
feat: add Bitbucket Cloud `your work` view

### ISSUE THREAD

**Issue by setchy** [setchy/atlassify#96]
### 📝 Provide a description of the new feature

Within Bitbucket there is a `Your Work` view, which shows any PRs that you have raised or have been asked to review.

Having this deeper-style "inbox" available via Atlassify would be userful to power users to keep track of their "work" view

**Comment by kadaan** [comment]
Would it be possible to get more granular notifications for Bitbucket PRs?  Fro instance a notification when someone comments on or pushed to a PR.

**Comment by setchy** [comment]
> Would it be possible to get more granular notifications for Bitbucket PRs? Fro instance a notification when someone comments on or pushed to a PR.

Unlikely. Those events currently aren't part of the Atlassian Events api offering.

**Comment by kadaan** [comment]
Darn, that would be the most useful for me at least.

**Comment by kadaan** [comment]
> Unlikely to get those in the form of "notifications". Those are not currently supported via the Atlassian Events api offering.

Does the `pullrequest:updated` event notify when a commit is pushed to a branch that the PR is on?

**Comment by setchy** [comment]
Bitbucket workspace events would be the solution, however, Atlassify is currently stateless and simply presents a view of latest data fetched from the Atlassian GraphQL API.

**Comment by kadaan** [comment]
> Bitbucket workspace events would be the solution, however, Atlassify is currently stateless and simply presents a view of latest data fetched from the Atlassian GraphQL API.

Makes sense, and I can completely understand why you would not want to make it stateful.  Still love Atlassify and maybe one day...

**Comment by setchy** [comment]
One day 🤞 😄
